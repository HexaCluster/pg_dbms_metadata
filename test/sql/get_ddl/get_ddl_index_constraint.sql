SELECT dbms_metadata.get_ddl('INDEX','perf_index','gdmmm');

SELECT dbms_metadata.get_ddl('INDEX','Gin_Index_Name','gdmmm');

SELECT dbms_metadata.get_ddl('CONSTRAINT','child_uniq','gdmmm');

SELECT dbms_metadata.get_ddl('CONSTRAINT','Unique age','gdmmm');

SELECT dbms_metadata.get_ddl('REF_CONSTRAINT','Fk_customer','gdmmm');

-- tests for schema as NULL
SET search_path TO public, gdmmm;

SELECT dbms_metadata.get_ddl('INDEX','perf_index');

SELECT dbms_metadata.get_ddl('INDEX','Gin_Index_Name');

SELECT dbms_metadata.get_ddl('CONSTRAINT','child_uniq');

SELECT dbms_metadata.get_ddl('CONSTRAINT','Unique age');
