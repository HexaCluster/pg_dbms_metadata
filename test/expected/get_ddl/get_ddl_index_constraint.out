                          get_ddl                           
------------------------------------------------------------
 CREATE INDEX perf_index ON gdmmm.table_all USING btree (c)
(1 row)

                                 get_ddl                                 
-------------------------------------------------------------------------
 ALTER TABLE gdmmm.table_all_child ADD CONSTRAINT child_uniq UNIQUE (id)
(1 row)

